cloudygames/models.py

```diff
@@ -8,15 +18,68 @@ class Game(models.Model):
     address = models.CharField(max_length=45)
     users = models.ManyToManyField(User)
 
+class GameSession(models.Model):
+    player = models.ForeignKey(User)
+    game = models.ForeignKey(Game)
+    controller = models.IntegerField()
+
+    def joinGame(self, _game):
```

Comment:
consider putting this into the [Manager methods](https://docs.djangoproject.com/en/1.8/topics/db/managers/#adding-extra-manager-methods). It is good to have this method to return GameSession object instead of controller id because calling join_game, I expect the game session. and i can also retrieve the controller from the game session if i want to know the controller later.